wdl-format/tests/format/complex_meta_and_calls/source.formatted.wdl

```diff
@@ -0,0 +1,119 @@
+version 1.0
+
+workflow test_wf {
+    meta {
+        a: "hello"
+        b: "world"
+        c: 5
+        d: -0xf
+        e: 1.0e10
+        f: -2.
+        g: true
+        h: false
+        i: null
+        j: {
+            a: [
+                1,
+                2,
+                3,
+            ],
+            b: [
+                "hello",
+                "world",
+                "!",
+            ],
+            c: {
+                x: 1,
+                y: 2,
+                z: 3,
+            },
+        }
+        k: [
+            {
+                a: {
+                },
+                b: 0,
+                c: "",
+                d: "",
+                e: [
+                ],
+            },
+            {
+                x: [
+                    1.0,
+                    2.0,
+                    3.0,
+                ],
+            },
+        ]
+    }
+
+    parameter_meta {
+        out_sj_filter_overhang_min: {
+            type: "SpliceJunctionMotifs",
+            label: "Minimum overhang required to support a splicing junction",
+        }
+    }
+
+    input {
+        SpliceJunctionMotifs out_sj_filter_overhang_min = SpliceJunctionMotifs {
+noncanonical_motifs: 30,
+GT_AG_and_CT_AC_motif: 12,
+}
```

Comment:
Are we missing indentation here?

Reply:
This is a consequence of `Expr` not being formatted (yet). So it's a known bug. I think we should merge this PR without `Expr` formatting, just to keep the PRs a more manageable length. We probably don't want to push a `sprocket` release until the `Expr` PR closes.